plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -100,6 +100,18 @@
     type: bool
     default: false
     version_added: 3.7.0
+  hostinterface_config:
+    required: false
+    description:
+      - setting dict of HostInterface on OOB controller
+    type: dict
+    version_added: '3.9.0'
```

Comment:
```suggestion
    version_added: '4.1.0'
```
The next release accepting features will be 4.1.0 (same below).